### PR TITLE
Mobx refind help

### LINKED ISSUE #4449: DE UX Testing | Onboarding v3: help discoverability "re-find help"
Covering issue no. 5 of [spreadsheet](https://docs.google.com/spreadsheets/d/10eWTU44vUEk-NKm2UcUPEK7qeHSDnRzrXUbMV6nDuBE/edit#gid=0) (medium priority):

**Issue**
> 5: "It was diifficult to return to Help content from Welcome Message once Tour is finished as all Help content is closed once the user selects 'Finish'.
One participant mentions it seems weird to click 'I'll need some help' until she's had a play around - this highlights how important it is that it is easy to return to Help content"

**Fixes**
https://projects.invisionapp.com/share/Z8XQ296K923
("OK got it" re-imagined)

### DoD:
- [x] Unit test written (if applicable)
- [x] Works on mobile (or deliberately disabled for mobile)
- [x] Code/peer reviewed (if didn't pair)
- [x] Relevant docs are updated
- [x] Previewable/deployed to dev (see separation of deployments)
- [x] Previewable/deployed to test (see separation of deployments)
- [x] (If a map-specific ticket) Issue linked to a release ticket

### Acceptance Criteria:
- [x] Help button location is highlighted when welcome message is closed
- [x] In line with welcome message being a config option, it should only appear when welcome message has been turned on in config

### PR DESCRIPTION
**Pull request by reginapramesti**
### What this PR does

Fixes #4449 

- Created a prompt to show users the location of help button, using the same prompt that is used for stories, just with different content.
- Refactored prompt component (replaced sass with styled components)
- Fixed minor bug: story prompt was previously off screen with the new UI buttons
- Fixed minor bug: helpful hints being cut off on smaller screens

### Checklist

-   [x] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

File: lib/ReactViews/SidePanel/SidePanel.jsx [hunk 1]
```diff
@@ -44,7 +44,14 @@ function EmptyWorkbench(props) {
         </Text>
       </Box>
       <Spacing bottom={10} />
-      <Box column paddedRatio={3}>
+      <Box
+        column
+        paddedRatio={3}
+        positionAbsolute
```
Comment (soyarsauce) [inline lib/ReactViews/SidePanel/SidePanel.jsx]: You'll need some responsive-to-height styles for this as you can end up with this case
image

Comment (reginapramesti) [reply lib/ReactViews/SidePanel/SidePanel.jsx]: Hmm should the helpful hints just be turned off at that point? It's either going to get clipped to like the first line or not even be visible

Comment (soyarsauce) [reply lib/ReactViews/SidePanel/SidePanel.jsx]: Could just make it overflowy: scroll like it does when workbench has long list of items

Comment (soyarsauce) [reply lib/ReactViews/SidePanel/SidePanel.jsx]: overflow-y:auto rather

Comment (reginapramesti) [reply lib/ReactViews/SidePanel/SidePanel.jsx]: Ok so I made the "Your workbench is empty" text responsive as well!

Comment (reginapramesti) [reply lib/ReactViews/SidePanel/SidePanel.jsx]: It didn't look very nice with the scroll bar 😬

**Comment by reginapramesti** [comment]
~~Made the entire `EmptyWorkbench` part responsive, but there are some things hardcoded which will need fixing if that's alright?~~ still discovered issues with overflow so will just resort to `overflow-y:auto` 😓

File: lib/ReactViews/SidePanel/SidePanel.jsx (hunk 1 above)
Comment (reginapramesti) [reply lib/ReactViews/SidePanel/SidePanel.jsx]: I just realised I only debugged when I had inspect open, so it still didn't fit well if I zoom in to 150% with the entire screen........... I think I might have to resort to overflow-y as a quick fix and let vic do the design for tiny screens

**Review by soyarsauce — APPROVED** [review]

**Pull request merged**